Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -94,7 +94,7 @@ The following part explains the configuration file and options:
       languageId: '2'
       title: Deutsch
       navigationTitle: ''
-      base: 'https://www.beispiel.de/'
+      base: 'https://example.de/'
```

Comment:
```suggestion
      base: 'https://example.net/'
```

Reply:
As this refers to "Deutsch" i would stick here to a specific `example.de` instead of a common `example.net`.

Reply:
> True but example.de does not exist and is none of the registered IANA example domains. If one registers example.de, we are in for trouble again.
Since the whole point of these changes is to normalize the examples to use official example domains we should stick to these.

Ok.